pythonforandroid/python.py

```diff
@@ -266,16 +267,19 @@ def create_python_bundle(self, dirn, arch):
                 '2' if self.version[0] == '2' else '',
                 self.major_minor_version_string
             ))
-        module_filens = (glob.glob(join(modules_build_dir, '*.so')) +
-                         glob.glob(join(modules_build_dir, '*.py')))
+        module_filens = list(glob.glob(join(modules_build_dir, '*.so')) +
+                             glob.glob(join(modules_build_dir, '*.py')))
+        info("Copy {} files into the bundle".format(len(module_filens)))
         for filen in module_filens:
-            shprint(sh.cp, filen, modules_dir)
+            info(" - copy {}".format(filen))
+            copy2(filen, modules_dir)
 
         # zip up the standard library
         stdlib_zip = join(dirn, 'stdlib.zip')
         with current_directory(join(self.get_build_dir(arch.arch), 'Lib')):
-            stdlib_filens = walk_valid_filens(
-                '.', self.stdlib_dir_blacklist, self.stdlib_filen_blacklist)
+            stdlib_filens = list(walk_valid_filens(
```

Comment:
This is perhaps the same thing as with `glob.glob()`.

Reply:
This one is not the same, we need that because `walk_valid_filens` it's a generator and we add a `info` message later where we print the number of files that we will zip using `len`...here we need a list...or remove the message that @tito added:

`info("Zip {} files into the bundle".format(len(stdlib_filens)))`

I think that this may be useful...what do you think?

Reply:
Yeah, for this use-case it makes sense.